ssl/record/rec_layer_s3.c

```diff
@@ -1007,20 +1007,21 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
         SSL3_RECORD_set_length(thiswr, len);
     }
 
+    wb = &s->rlayer.wbuf[0];
```

Comment:
This is not correct in the case that `numpipes > 1`. Each pipe uses a different buffer.

Reply:
but here it looks like wbuf[] is an array:
https://github.com/openssl/openssl/blob/f868abcc5dbcbed6ca2e33bdb9bf06c817a4cce3/ssl/record/rec_layer_s3.c#L799-L802

I want to pass that array on to the enc function.  that is the easiest way I could find to get it informed, and that does also work for dtls.

Reply:
Yeah, sorry I misread the code here